### PR TITLE
[Ready for Review] Add welcome bot

### LINKED ISSUE #860: [WIP] Celebration Bot! Auto-comment on PRs opened by first-time contributors to thank them
### Summary

Add a GitHub Action to the repository that automatically comments on Pull Requests opened by new/first-time contributors congratulating them and welcoming them to the project.

### What needs to be done?

- [ ] Write the relevant GitHub Action

### Who can help?

* @nathanbegbie 

---

### Updates

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### PR DESCRIPTION
**Pull request by sgibson91**
### Summary

Fixes #860 

This pull request is configuring the [behaviorbot/welcome](https://github.com/behaviorbot/welcome) probot app on the repo. This bot will leave a comment when first-time contributors open a new issue/pull request and successfully merge their first pull request. The idea is to make them feel welcomed to the community and provide any additional info on where they can get help if they need it.

### List of changes proposed in this PR (pull-request)

* Added `.github/config.yml` containing the welcome bot config

### What should a reviewer concentrate their feedback on?

- [x] Is the wording ok?
- [x] Do the links work?
- [x] Do the images render correctly?
- [x] Everything looks ok?

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by sgibson91** [comment]
The banners work! (I was unable to test the issue comment as you can't create issues on a fork of a repo)

The links need correcting and translating the text from YAML to Markdown formatting also needs some work.

Screenshot 2020-05-06 at 13 26 37

Screenshot 2020-05-06 at 13 27 29

**Comment by sgibson91** [comment]
How I tested the bot:

1) Forked the repo to my namespace
2) made the `add-welcome-bot` branch the default branch on my fork
3) Created a new GitHub account
4) Forked sgibson91's fork into the new account's namespace
5) Made a change and opened the pull request as the new account
6) Merged the PR into sgibson91's fork as myself (because write access)

**Comment by sgibson91** [comment]
Getting there with the Markdown formatting

Screenshot 2020-05-06 at 13 57 07

**Comment by sgibson91** [comment]
I think this is the best I can achieve with the formatting

Screenshot 2020-05-06 at 14 14 53

**Comment by nathanbegbie** [comment]
@sgibson91 this is amazing! I've been fiddling around with GraphQL requests to try and get this right with a GH Action. . . and this app was here all along 🙈 Super glad this has happened!

**Comment by sgibson91** [comment]
Haha, it seems there is a probot app for everything nowadays! Except one that fixes the bugs and implements the features of course 😉

**Comment by malvikasharan** [comment]
@sgibson91 Thank you for this amazing Bot. I am currently testing this on my fork and will add my review soon.

**Comment by malvikasharan** [comment]
Tested it on Autistica project. Looks fantastice! I am approving this, however, I think we should add the image citation: "The Turing Way Community, & Scriberia. (2019, July 11). Illustrations from the Turing Way book dashes. Zenodo. http://doi.org/10.5281/zenodo.3332808".

What do you think?

**Comment by sgibson91** [comment]
Hi @malvikasharan. I made the images into links to the Zenodo record as I didn't want the comments the bot leaves to be overly-verbose and confusing to new contributors 😄Let me know what you think!

**Comment by malvikasharan** [comment]
Thanks @sgibson91. I am happy with the Zenodo link. 
I will add a line in readme stating that the Illustration with <citation> has been used in the welcome-bot response.

**Comment by malvikasharan** [comment]
Can you please review the section in README? Once we finalise that, you can merge this PR. 
Once again, thank you so much. This is super cool! 😎

File: README.md [hunk 1]
```diff
@@ -76,6 +76,20 @@ You might want to include a web archive link such as: <https://web.archive.org/w
 We really appreciate any references that you make to _The Turing Way_ project in your and we hope it is useful.
 If you have any questions please [get in touch](#get-in-touch).
 
+### Citing _The Turing Way_ illustrations
+
+_The Turing Way_ llustrations are created by Scriberia artists as part of [_The Turing Way book dashes](https://github.com/alan-turing-institute/the-turing-way/tree/master/workshops/book-dash) in Manchester on 17 May 2019, and London on 28 May 2019 and 21 February 2020.
```
Comment (sgibson91) [inline README.md:81]:
```suggestion
_The Turing Way_ illustrations are created by artists from [Scriberia](https://www.scriberia.co.uk/) as part of [_The Turing Way_ book dashes](https://github.com/alan-turing-institute/the-turing-way/tree/master/workshops/book-dash) in Manchester on 17 May 2019, and London on 28 May 2019 and 21 February 2020. 
```

File: README.md [hunk 2]
```diff
@@ -76,6 +76,20 @@ You might want to include a web archive link such as: <https://web.archive.org/w
 We really appreciate any references that you make to _The Turing Way_ project in your and we hope it is useful.
 If you have any questions please [get in touch](#get-in-touch).
 
+### Citing _The Turing Way_ illustrations
+
+_The Turing Way_ llustrations are created by Scriberia artists as part of [_The Turing Way book dashes](https://github.com/alan-turing-institute/the-turing-way/tree/master/workshops/book-dash) in Manchester on 17 May 2019, and London on 28 May 2019 and 21 February 2020. 
+They depict a variety of content of the handbook, collaborative efforts in the community and The Turing Way project in general.
```
Comment (sgibson91) [inline README.md:82]:
```suggestion
They depict a variety of content from the handbook, collaborative efforts in the community and _The Turing Way_ project in general. 
```

File: README.md [hunk 3]
```diff
@@ -76,6 +76,20 @@ You might want to include a web archive link such as: <https://web.archive.org/w
 We really appreciate any references that you make to _The Turing Way_ project in your and we hope it is useful.
 If you have any questions please [get in touch](#get-in-touch).
 
+### Citing _The Turing Way_ illustrations
+
+_The Turing Way_ llustrations are created by Scriberia artists as part of [_The Turing Way book dashes](https://github.com/alan-turing-institute/the-turing-way/tree/master/workshops/book-dash) in Manchester on 17 May 2019, and London on 28 May 2019 and 21 February 2020. 
+They depict a variety of content of the handbook, collaborative efforts in the community and The Turing Way project in general. 
+These illustrations are available on Zenodo: https://zenodo.org/record/3695300 under a CC-BY license.
```
Comment (sgibson91) [inline README.md:83]:
```suggestion
These illustrations are available on Zenodo ([https://zenodo.org/record/3695300)](https://zenodo.org/record/3695300) under a CC-BY license.
```

File: README.md [hunk 4]
```diff
@@ -76,6 +76,20 @@ You might want to include a web archive link such as: <https://web.archive.org/w
...
+
+When using any of the images, please include the following attribution:
+> This image was created by Scriberia for The Turing Way community and is used under a CC-BY licence.
+
+The latest version from Zenodo can be cited as:
+> The Turing Way Community, & Scriberia. (2020, March 3). Illustrations from the Turing Way book dashes. Zenodo. http://doi.org/10.5281/zenodo.3695300
+
+We have used a few of these illustrations in [Welcome Bot](https://github.com/apps/welcome) responses to the new members' contributions on this GitHub.
```
Comment (sgibson91) [inline README.md:91]:
```suggestion
We have used a few of these illustrations in the [Welcome Bot](https://github.com/apps/welcome)'s responses to new members' contributions in this GitHub repository.
```

**Comment by malvikasharan** [comment]
Thanks @sgibson91! With that we are ready to merge this :)

**Comment by sgibson91** [comment]
Yes! 🎉

**Pull request merged**